.circleci/config.yml

```diff
@@ -53,6 +55,19 @@ jobs:
       - checkout
       - run: pip install yamllint
       - run: yamllint -d .yamllint.yml .
+  shellcheck:
+    docker:
+      - image: cimg/base:2020.11
+    steps:
+      - checkout
+      - shellcheck/install
+      - shellcheck/check
+  audit:
+    docker:
+      - image: cimg/node:14.15
+    steps:
+      - checkout
+      - run: npm audit
```

Comment:
is there a reason to re-add audit? I've had it before but removed it because it may fail because of the conditions not related to the PR itself, and IMO PR checks should be isolated

Reply:
Yeah, I actually wanted it to just run once in a while but didn't find a good way to do this. Let me know if you have any ideas on this.
And audit should pass most of the time anyway with dependabot.
I was thinking of making it optional, so we would still merge it even if audit fails.

Reply:
Okay, I've changed it so it only runs on master. Would that be good for you?

Reply:
I'd rather run it once a week or something if possible and not when pushing to any branch (as it may fail because of the external factors), not sure if it's possible on CircleCI though

Reply:
https://circleci.com/docs/2.0/configuration-reference/#schedule this may work

Reply:
I think that might indeed work, not sure why I did not find that before. I will check it out and change it

Reply:
one last thing, does it send the result somewhere? if not I'd suggest also adding a slack notification (like the one in "docker build and push" step)

Reply:
We should get an email notification if it fails, but I can set up a Slack notification as well if you'd like

Reply:
I'd say this is optional but would be nice to have